hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java

```diff
@@ -174,12 +174,13 @@ private static void collectionXmlGenerator(XmlGenerator gen, String type, Collec
     private static void replicatedMapConfigXmlGenerator(XmlGenerator gen, Config config) {
         for (ReplicatedMapConfig r : config.getReplicatedMapConfigs().values()) {
             gen.open("replicatedmap", "name", r.getName())
-                    .node("in-memory-format", r.getInMemoryFormat())
-                    .node("concurrency-level", r.getConcurrencyLevel())
-                    .node("replication-delay-millis", r.getReplicationDelayMillis())
-                    .node("async-fillup", r.isAsyncFillup())
-                    .node("quorum-ref", r.getQuorumName())
-                    .node("statistics-enabled", r.isStatisticsEnabled());
+               .node("in-memory-format", r.getInMemoryFormat())
```

Comment:
Could you please revert this block up to the `// TODO`? It's already fixed in https://github.com/hazelcast/hazelcast/pull/12196